multipaz/src/commonMain/kotlin/org/multipaz/mdoc/transport/BleTransportPeripheralMdoc.kt

```diff
@@ -64,6 +63,7 @@ internal class BleTransportPeripheralMdoc(
         peripheralManager.setCallbacks(
             onError = { error ->
                 runBlocking {
+                    currentJob?.cancel("onError was called", error)
```

Comment:
I noticed that this cancellation was missing from the peripheral server mode transports. I wasn't 100% sure if it was intentional or not, but it looks like it has been just forgotten from here (and from the reader counterpart).